components/automate-ui/src/app/components/table/table-body/table-body.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, HostBinding } from '@angular/core';
+
+@Component({
+  selector: 'chef-table-body',
+  templateUrl: './table-body.component.html',
+  styleUrls: ['./table-body.component.scss']
+})
+export class TableBodyComponent {
+  @HostBinding('attr.role') role = 'rowgroup';
```

Comment:
Similarly, the roles provide accessibility semantics for tables (see https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Table_Role)

Reply:
Thanks for thinking of accessibility here 🥇